From f3547f072b2737b08098404fed40a12e6ede498b Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 5 Dec 2024 15:20:48 +0800
Subject: [PATCH 181/273] AOSCOS: drm: radeon: declare prototypes for
 radeon_lvds_dpms_{off,on}()

Declare prototypes for radeon_lvds_dpms_off() and radeon_lvds_dpms_on()
so that the `extern' references would work in the lemote3a-laptop platform
driver.

Fixes: "AOSCOS: MIPS: Loongson: Add a hook to turn on/off DPMS when Fn+F7 is pressed"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/gpu/drm/radeon/radeon_drv.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/gpu/drm/radeon/radeon_drv.h b/drivers/gpu/drm/radeon/radeon_drv.h
index 02a65971d140..a5bc85facefd 100644
--- a/drivers/gpu/drm/radeon/radeon_drv.h
+++ b/drivers/gpu/drm/radeon/radeon_drv.h
@@ -136,4 +136,8 @@ static inline bool radeon_has_atpx_dgpu_power_cntl(void) { return false; }
 static inline bool radeon_is_atpx_hybrid(void) { return false; }
 #endif
 
+/* DPMS handlers */
+void radeon_lvds_dpms_off(void);
+void radeon_lvds_dpms_on(void);
+
 #endif				/* __RADEON_DRV_H__ */
-- 
2.47.1

